src/Apps/Consign/Routes/MarketingLanding/MarketingLandingApp.tsx

```diff
@@ -34,7 +42,21 @@ export const MarketingLandingApp = () => {
         sessionStorage.setItem("utmParams", JSON.stringify(queryUtmParams))
       }
     }
-  }, [])
+  }, [query.utm_medium, query.utm_source, query.utm_term])
```

Comment:
fyi: couldn't commit without this change

Reply:
FYI: you can always disable exhaustive deps rule for that line.